client/components/theme-collection/use-theme-collection.ts

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'calypso/state';
+import { getThemesForQueryIgnoringPage } from 'calypso/state/themes/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+interface ThemesQuery {
+	page: number;
+	number: number;
+	tier: string;
+	filter: string;
+	search: string;
+	collection: string;
```

Comment:
Is `collection` actually a query property?

Reply:
yup